tests/routers/test_paths.py

```
@@ -0,0 +1,179 @@
+    db_session.commit()
+
+    response = authenticated_client.get(f"/api/v1/paths/{path.id}/scan-errors")
+    assert response.status_code == 200
+    data = response.json()
+    assert data["last_scan_status"] == "FAILURE"
+    assert data["last_scan_error_log"] == "Disk full"
+
+@patch("shutil.disk_usage")
+def test_get_hot_storage_stats(mock_disk_usage, authenticated_client: TestClient, monitored_path_factory, tmp_path):
+    """Test the /stats endpoint."""
+    path = monitored_path_factory("Stats Path", str(tmp_path / "stats_hot"))
```

Ruff (0.14.14)

[error] Local variable `path` is assigned to but never used

Remove assignment to unused variable `path`

(F841)